test/configuration.go

```diff
@@ -31,7 +29,7 @@ import (
 type Options struct {
 	EnvVars              []corev1.EnvVar
 	ContainerConcurrency int
-	RevisionTimeout      time.Duration
+	RevisionTimeout      int64
```

Comment:
We should either leave this a `time.Duration` and convert when needed or qualify this as `RevisionTimeoutSeconds`.

Reply:
Done.